src/lv_colors.py

```diff
@@ -0,0 +1,24 @@
+import lvgl as lv
+def LV_COLOR_MAKE(r,g,b):
+#    return lv.color_hex(r<<16| g<<8 |b)
+    return lv.color_make(r,g,b)
+
+class lv_colors:
```

Comment:
So `lv_colors.py ` can probably be removed from here, because it can be [imported from lib](https://github.com/lvgl/lv_binding_micropython/blob/master/lib/lv_colors.py).

Reply:
removed it